scripts/ci-entrypoint.sh

```diff
@@ -141,6 +141,16 @@ wait_for_nodes() {
     "${KUBECTL}" get nodes -owide
 }
 
+copy_secret() {
+    unset KUBECONFIG
+    "${KUBECTL}" get secret "${CLUSTER_NAME}-control-plane-azure-json" -o jsonpath='{.data.control-plane-azure\.json}' | base64 --decode > azure_json
+    
+    export KUBECONFIG="${KUBECONFIG:-${PWD}/kubeconfig}"
+    kubectl create secret generic "${CONFIG_SECRET_NAME}" -n kube-system \
```

Comment:
```suggestion
    "${KUBECTL}" create secret generic "${CONFIG_SECRET_NAME}" -n kube-system \
```
to make sure we're using the hack/tools/bin kubectl binary

Reply:
> to make sure we're using the hack/tools/bin kubectl binary

Thanks for correcting it! Updated.